src/control/server/config/server_test.go

```diff
@@ -189,7 +194,7 @@ func TestServerConfig_Constructed(t *testing.T) {
 		WithDisableVFIO(true).   // vfio enabled by default
 		WithEnableVMD(true).     // vmd disabled by default
 		WithEnableHotplug(true). // hotplug disabled by default
-		WithNrHugePages(4096).
+		WithNrHugePages(0).      // hugepages auto-calculated by default
```

Comment:
Maybe would be better to remove this line, if we basically want the default?

Reply:
can do, was just being explicit

Reply:
done